project/Newevaluation.html

```diff
@@ -6,85 +6,118 @@
   <style>
     body{
       background-color: rgb(82, 86, 89);
-      margin-top: 50px;
+      margin-top: 3%;
       margin-left: 0;
     }
 
+
+
+/* ---------------------------------Navigation Bar--------------------------- */
+
     .active{
       border-bottom: 6px solid DodgerBlue;
-      padding:18px;
+      padding: 1%;
     }
 
     nav {
       display: inline;
       background-color: whitesmoke;
-      width: 1000px;
-      height: 100px;
+      width: 100%;
+    //  height: 70%;
```

Comment:
Ah, but is this a comment? Or is this not a comment? Do note that this is CSS. :)